readthedocs/proxito/middleware.py

```diff
@@ -144,6 +145,8 @@ def process_response(self, request, response):  # noqa
 
         hsts_header_values = []
 
+        self.add_proxito_headers(request, response)
```

Comment:
Rather than adding a single-use mixin in a separate file, couldn't this method just be a method on this class?

Reply:
Yea, I thought we'd have to use it more places, but we haven't yet, so that seems fine. 👍